doc/stdlib/index-list.html.template

```diff
@@ -705,7 +705,7 @@ through the <tt>Require Import</tt> command.</p>
     theories/Compat/Coq818.v
     theories/Compat/Coq819.v
     theories/Compat/Coq820.v
-    theories/Compat/Coq821.v
+    theories/Compat/Coq90.v
```

Comment:
Sorry that I didn't review this before Jason adapted his tools, but this should be `Rcoq90`, not `Coq90`.